lib/beaker-pe/install/pe_utils.rb

```diff
@@ -285,9 +285,14 @@ def fetch_pe(hosts, opts)
             # We install Puppet from the master for frictionless installs, so we don't need to *fetch* anything
             next if host['roles'].include?('frictionless') && (! version_is_less(opts[:pe_ver] || host['pe_ver'], '3.2.0'))
 
-            if host['platform'] =~ /windows/
-              fetch_pe_on_windows(host, opts)
-            elsif host['platform'] =~ /osx/
+            # Windows agents install directly over HTTP and Windows can't run master
+            next if host['platform'] =~ /windows/
```

Comment:
This may be heavy-handed.  Might want to keep the logic in generic_install, as I haven't checked everything that calls fetch_pe.